src/day-1-toy/args.py

```diff
@@ -4,14 +4,25 @@
 # Write a function f1 that takes two integer positional arguments and returns
 # the sum. This is what you'd consider to be a regular, normal function.
 
-#def f1(...
+def f1(x, y):
+    return (x + y)
 
 print(f1(1, 2))
 
 # Write a function f2 that takes any number of iteger arguments and prints the
 # sum. Google for "python arbitrary arguments" and look for "*args"
 
-# def f2(...
+def f2(*args):
+    z = 0
+    for i in args:
+        if isinstance(i, list):
+            for j in i:
+                z += j
+        else:
+            z += i
+    return z
+
```

Comment:
There's some real interesting stuff going on here. Can you walk me through this bit?

Reply:
this was before I realized you can put * in the input when calling a function, so i had to find a way to read my list items INCLUDING a list as a list item